src/nvim/lua/stdlib.c

```diff
@@ -536,11 +536,9 @@ static int nlua_iconv(lua_State *lstate)
   return 1;
 }
 
-// Like 'zx' but don't call newFoldLevel()
 static int nlua_foldupdate(lua_State *lstate)
```

Comment:
Without the comment, it's not clear what the purpose of this function is. Do we still need `vim._foldupdate()` or can `zx` be used?

Reply:
Above info is worth adding to the commit message.

> `zx` has the additional side effect of opening the folds under cursor,

Seems like that is worth a 1-line code comment on the function.

Reply:
Added to the comment